spring/boot2-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaReactiveWebServerFactory.java

```diff
@@ -259,7 +260,9 @@ private void configureArmeriaService(ServerBuilder sb, DocServiceBuilder docServ
         configurePorts(sb, settings.getPorts());
         configureServerWithArmeriaSettings(sb, settings,
                                            firstNonNull(findBean(MeterRegistry.class), Metrics.globalRegistry),
-                                           findBeans(HealthChecker.class));
+                                           findBeans(HealthChecker.class),
+                                           firstNonNull(findBean(MeterIdPrefixFunction.class),
+                                                        MeterIdPrefixFunction.ofDefault("armeria.server")));
```

Comment:
Could you use a ternary operator(`? : `) instead of `firstNonNull` to avoid creating an extra object?